apps/client/src/app/services/token-storage.service.ts

```diff
@@ -11,23 +11,23 @@ export class TokenStorageService {
   public constructor(private userService: UserService) {}
 
   public getToken(): string {
-    return window.localStorage.getItem(TOKEN_KEY);
+    return window.sessionStorage.getItem(TOKEN_KEY);
   }
 
   public saveToken(token: string): void {
-    window.localStorage.removeItem(TOKEN_KEY);
-    window.localStorage.setItem(TOKEN_KEY, token);
+    window.sessionStorage.removeItem(TOKEN_KEY);
+    window.sessionStorage.setItem(TOKEN_KEY, token);
   }
 
   public signOut(): void {
-    const utmSource = window.localStorage.getItem('utm_source');
+    const utmSource = window.sessionStorage.getItem('utm_source');
```

Comment:
This has to be kept in `localStorage`!

Reply:
If you delete the `localStorage`, you have to save it temporarily and add it again

Reply:
Hmmm and if I no longer clear localStorage? It's weird because I need to keep track of the webauthn ids. Or should a sign out also clear those?

Reply:
If you don't clear it, you don't have to keep `utm_source` neither in localStorage nor sessionStorage 😃